packages/x-components/src/__stubs__/popular-searches-stubs.factory.ts

```diff
@@ -0,0 +1,34 @@
+import { Suggestion } from '@empathyco/x-types';
+
+/**
+ * Creates a list of generic popular searches.
+ *
+ * @param amount - The number of popular searches to create.
+ *
+ * @returns An array of popular searches.
+ */
+export function getPopularSearchesStub(amount = 3): Suggestion[] {
+  return Array.from({ length: amount }, (_, index) =>
+    createPopularSearch(`Popular search ${index}`)
+  );
+}
+
+/**
+ * Creates a popular search stub with the provided options.
+ *
+ * @param query - The query of the suggestion.
+ * @param suggestion - An optional object with fields to override the suggestion.
+ *
+ * @returns A popular search.
+ */
+export function createPopularSearch(query: string, suggestion?: Partial<Suggestion>): Suggestion {
```

Comment:
Can we re-use the `createQuerySuggestion` method adding the modelName as parameter and remove this one?

Reply:
No because of the missunderstandings. If you want to create a query suggestion you can use the `createQuerySuggestionStub`, if you want to create a  popular search you can use the `createPopularSearchStub`. Passing the model name from the beginning was a bad idea because it allowed you to create invalid models.